corehq/apps/hqadmin/static/hqadmin/js/project_map.js

```diff
@@ -132,16 +135,28 @@ var projectMapInit = function(mapboxAccessToken) {
     });
 
     function getColor(featureId) {
+        var thresholdScales;
         var count = dataController.getCount(featureId);
-        if (!count) {
-            return COUNTRY_COLORS[0];
+        var isProjectCountMap = dataController.isProjectCountMap();
+        if (isProjectCountMap) {
+            thresholdScales = PROJECT_COUNTS_THRESHOLD;
+        } else {
+            thresholdScales = USER_COUNTS_THRESHOLD;
         }
-        var pct = count / dataController.getMax();
-        var index = Math.min(Math.floor(pct * COUNTRY_COLORS.length), COUNTRY_COLORS.length - 1);
-
+        var index = getColorScaleIndex(count, thresholdScales);
         return COUNTRY_COLORS[index];
     }
 
+    function getColorScaleIndex(count, scales) {
+        for (var i in scales){
+            if (count < scales[i]){
+                return i;
+            } else {
+                return scales.length;
```

Comment:
I think this statement should be outside of the for loop. Suppose that count was 15 and scales was `[10, 20, 30]`. On the first iteration of the loop, the if statement would be false, so then it would return 3 (`scales.length`), but I think the right return value would be 2.